### DISCUSSION TITLE
[Run issue]:/examples/geometry/nucleusModel/Nucleus.txt

### DISCUSSION THREAD

**Discussion by nknanfeng** [topas-nbio/TOPAS-nBio#28] [Run-Problem] [answered]
### What was the issue?

I hope this message finds you well.

I am currently working with the /examples/geometry/nucleusModel/Nucleus.txt example from TOPAS-nBio. I am encountering issues with running this example on my personal computer and am seeking guidance on the required system specifications for optimal performance.

Could you please provide the following details:

Computer Configuration: What are the recommended hardware specifications (CPU, RAM, GPU, etc.) for running this example efficiently?
Performance Details: What was the computer configuration used during your experiments, and how long did it take to complete the task?
Understanding these requirements will greatly assist me in troubleshooting the performance issues I am experiencing and achieving the desired output.

Thank you for your assistance. I look forward to your response.

Best regards,

### TOPAS-nBio category

Geometry

### Version

3.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by Geoffvinc** [comment]
Hi Nanfeng,

can you elaborate more on the performance issue that you encountered and whether if you made any modification to the original example script (e.g. enabling the visualization)?

I don't know about the specific system specification for it. However, I advise you to use multiple thread to run the simulation by including this in the script.
`i:Ts/NumberOfThreads = 10`

Regards,
Geoff

**Reply by nknanfeng** [reply]
Hi Geoff,
Thank you very much for your reply. I did not make any modifications to the original example script. There were originally visualization-related settings at the end of the source file.
My computer configuration is as follows: CPU: 24 cores and 32 threads. Memory: 32GB SSD. You mentioned that multithreading can be set. Is it because the complexity of the nucleus model is very high and it takes a long time to get results?
Regards,
Nanfeng

**Reply by Geoffvinc** [reply, marked as answer]
Yes, there is visualisation settings in the example script. make sure that the settings is turned off by adding "#" at the start of each line.

Your computer configuration seems to be more than sufficient to run the simulation. You are correct, the nucleus model and its scoring is very complex (contains 6.078*10^9 bp of DNA geometry) and it takes long time to get the result, especially for particle with low energy and high number of histories setup. Therefore, multithreading is recommended.

Regards,
Geoff

**Reply by nknanfeng** [reply]
Thank you very much for your reply. It is of great help to me. Now I will try again according to your suggestions.
Regards,
Nanfeng
